Resources/Prototypes/_CM14/Entities/Objects/Tools/tools.yml

```
@@ -79,3 +79,30 @@
     state: red-icon
   - type: Item
     heldPrefix: red
+
+- type: entity
+  parent: BaseItem
+  id: CMHandLabeler
+  name: hand labeler
+  description: A hand labeler, used to label items and objects.
+  components:
+    - type: Sprite
+      sprite: _CM14/Objects/Tools/hand_labeler.rsi
+      state: hand_labeler
+    - type: Item
+      sprite: _CM14/Objects/Tools/hand_labeler.rsi
+    - type: UseDelay
+      delay: 2.0
+    - type: ActivatableUI
+      key: enum.HandLabelerUiKey.Key
+      inHandsOnly: true
+    - type: UserInterface
+      interfaces:
+        - key: enum.HandLabelerUiKey.Key
+          type: HandLabelerBoundUserInterface
+    - type: HandLabeler
+      whitelist:
+        components:
+          - Item
+        tags:
+          - Structure
```

By parenting from ss14's `HandLabeler` you can remove 90% of this copied stuff
```suggestion
  parent: [ HandLabeler , CMEntityBase ]
  id: CMHandLabeler
  components:
    - type: Sprite
      sprite: _CM14/Objects/Tools/hand_labeler.rsi
      state: hand_labeler
    - type: Item
      sprite: _CM14/Objects/Tools/hand_labeler.rsi
```